audio_service/lib/audio_service.dart

```diff
@@ -628,13 +652,12 @@ class AudioService {
   /// The root media ID for browsing the most recently played item(s).
   static const String recentRootId = 'recent';
 
-  // ignore: close_sinks
-  static final BehaviorSubject<bool> _notificationClickEvent =
-      BehaviorSubject.seeded(false);
-
   /// A stream that broadcasts the status of the notificationClick event.
   static ValueStream<bool> get notificationClickEvent =>
       _notificationClickEvent;
+  // ignore: close_sinks
```

Comment:
Do you have a reason for preferring to swap the order?

Reply:
i mostly follow flutter style guide and initially i was formatting code to match it
after discussion with you, i dropped this, but i forgot to revert some changes i made